app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ActiveRecord::Base
+  validates_presence_of   :provider, :uid, :login, :email, :token
```

Comment:
Stroring `provider` here may be unnecessary, the only auth this app will ever use is GitHub.